docs/api_marketplace.yaml

```diff
@@ -966,6 +1082,67 @@ components:
             States ceiling price, the maximum asking price. Rate is charged per
             hour.
 
+    Bid_put:
```

Comment:
I thought we only allowed updating the `status` field? If that's the case, I'm not convinced we should permit all these other fields. Thoughts?